qan/manager_test.go

```diff
@@ -204,19 +205,10 @@ func (s *ManagerTestSuite) TestStartWithConfig(t *C) {
 			UUID:           mysqlInstance.UUID,
 			CollectFrom:    analyzerType,
 			Interval:       300,
-			WorkerRunTime:  600,
-			MaxSlowLogSize: 100,  // specify optional args
```

Comment:
Those values changed, I don't know why and where they are defined but hey, now tests works